sdk/python/feast/repo_operations.py

```diff
@@ -325,13 +325,12 @@ def registry_dump(repo_config: RepoConfig, repo_path: Path) -> str:
     return json.dumps(registry_dict, indent=2, sort_keys=True)
 
 
-def cli_check_repo(repo_path: Path):
+def cli_check_repo(repo_path: Path, fs_yaml_file: Path):
```

Comment:
nitty but it's slightly confusing to me that `cli_check_repo` and `load_repo_config` have the args switched in order; is it possible to use the same order?

Reply:
sure makes sense